packages/orchestrator-ui-components/src/components/WFOSubscription/SubscriptionDetailTree.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
```

Comment:
Why isn't this name WFOSubscriptionDetailTree?

When working with the quick search in my IDE I know have to remember which components have the prefix

Reply:
same